src/views/studio/CreateMemberView/CreateMemberView.tsx

```diff
@@ -97,6 +104,16 @@ const CreateMemberView = () => {
     debounceAvatarChange(value)
   }
 
+  const debounceHandleChange = debounce(async (value) => {
+    clearErrors('handle')
+    getMember({ variables: { where: { handle: value } } })
+  }, 1000)
+
+  const handleHandleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
handleHandleChange ❤️  
handleMemberName should also work well I think 😅 , but not a big deal for me, you can leave it as it is now

Reply:
Yup, chuckled a little as well when writing this 😅

Reply:
I mean, it makes sense... 😅